File: src/main/java/project/xml/main_schema/TLice.java
package project.xml.main_schema;

import lombok.Getter;
import lombok.Setter;

import javax.xml.bind.annotation.*;

@XmlAccessorType(XmlAccessType.FIELD)
@XmlType(name = "TLice", propOrder = {
    "adresa",
    "kontakt"
})
@XmlSeeAlso({
    TPravnoLice.class,
    TFizickoLice.class
})
@Getter
@Setter
public abstract class TLice {

    @XmlElement(name = "Adresa", required = true)
    protected Adresa adresa;
    @XmlElement(name = "Kontakt", required = true)
    protected Kontakt kontakt;

}

File: XML_client/src/app/p1/model/t-lice.ts
import {Adresa} from "./adresa";
import {Kontakt} from "./kontakt";

export abstract class TLice{
  constructor(
    private ime:string,
    private prezime:string,
    private drzavljanstvo:string,
    private poslovnoIme:string,
    private adresa:Adresa,
    private kontakt:Kontakt
  ) {}
}
